lms/envs/common.py

```diff
@@ -4413,6 +4413,12 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 # .. toggle_creation_date: 2016-06-30
 ENABLE_COMPREHENSIVE_THEMING = False
 
+# .. setting_name: CUSTOM_RESOURCE_TEMPLATES_DIRECTORY
+# .. setting_default: None
+# .. setting_description: Path to existing directory containing resource templates.
+#   "CUSTOM_RESOURCE_TEMPLATES_DIRECTORY" : null
+CUSTOM_RESOURCE_TEMPLATES_DIRECTORY = []
+
```

Comment:
My understanding is that this is only ever needed in Studio, while creating content. 
Can you remove this from here? If not, can you explain why it needs to stay in a comment?

Reply:
Hmmm, I think you're right. I thought at the time that the LMS needed to load the yaml files as well for display, but it might not be needed.

Reply:
It's needed. I'm not sure where, but without it the tests fail to pass, because modules on the LMS side use this code as well.